pom.xml

```diff
@@ -70,7 +70,7 @@
     <dependency>
       <groupId>org.fusesource.jansi</groupId>
       <artifactId>jansi</artifactId>
-      <version>2.0.1</version>
+      <version>2.2.0</version>
```

Comment:
should this be 2.3.1 now?

Reply:
We could, but the 2.3.0/2.3.1 only brings two additional native methods specifically for `mvnd` and no other changes, so there is no real benefit either.